ReactNativeClient/lib/components/note-body-viewer.js

```diff
@@ -203,6 +210,33 @@ class NoteBodyViewer extends Component {
 		return this.forceUpdate_;
 	}
 
+	async onResourceLongPress(resourceId) {
+		const resource = await Resource.load(resourceId);
+		const name = resource.title ? resource.title : resource.file_name;
+
+		const action = await dialogs.pop(this, name, [
+			{ text: _('Share'), id: 'share' },
+		]);
+
+		if (action === 'share') {
+			const file_name = resource.file_name ?
```

Comment:
`filename` for consistency (db objects are not camel case but local variables should be)

Reply:
both done